src/config/default-app-config.ts

```diff
@@ -57,6 +57,18 @@ export class DefaultAppConfig implements AppConfig {
     nameSpace: '/',
   };
 
+  // The SSR REST API server settings, defaulting to rest.
+  // NOTE: these must be include in rest.cors.allowed.origins in your backend's local.cfg.
```

Comment:
An internal network URL could be `http://dspace-angular:4200` and the external web server URL could be `https://dspace.angular.edu`. Wont the UI server-side origin making requests to the API server-side from `http://dspace-angular:4200` while the browser origin making the requests from URL `https://dspace.angular.edu`?

Reply:
@wwelling : I might be missing your point, but `rest.cors.allowed.origins` is the list of *client* `Origin` headers which the REST API will allow to login, etc.

In this scenario, the SSR REST API settings are not a new *client*, they are just a different way of contacting the backend.  So, I don't think they result in the *client's* `Origin` having a different value.

However, you are correct that if the backend receives requests from two different clients (an external proxy URL and an internal SSR client), then both those clients may need to be added to `rest.cors.allowed.origins`.   However, I don't believe either of those clients will be using the new `ssr.rest.*` settings (as you said, the client will always either use the external URL or an internal server-side client ULR like http://dspace-angular:4000, and neither of those correspond to the `ssr.rest.*` settings).  

In summary, I don't believe the `ssr.rest.*` URL settings need to be added to `rest.cors.allowed.origins`, unless I'm missing your point.  But, you are correct that we may need to determine if the server-side client (like http://dspace-angular:4000) needs to be added to `rest.cors.allowed.origins` (I haven't tested that yet).

Reply:
Ah, I see the miscommunication. Yes, the comment was intended to be the clients associated with two different URL to make request to the server-side API. The comment can go but should be documented somewhere because the angular SSR client can have a different origin than the URL provided for the CSR client.